recipe/meta.yaml

```
@@ -12,17 +12,18 @@ source:
 
 build:
   noarch: python
```

```suggestion
  skip: true  # [py<39]
```
See https://github.com/arviz-devs/xarray-einstats/blob/4d9b8c76284e4e56f1691060b200dd4b83424382/pyproject.toml#L9